core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/console-reporter.ts

```diff
@@ -67,6 +67,9 @@ const CONSOLE_ERRORS_TO_IGNORE = [
       'the server responded with a status of 405 ()',
     'g'
   ),
+  // Error occurs due to ExpressionChangedAfterItHasBeenCheckedError.
+  // This can be removed after solving the issue #20189.
```

Comment:
Why #20189 is used twice, Firstly in `CONSOLE_ERRORS_TO_IGNORE` then in line 80 `CONSOLE_ERRORS_TO_FIX`
If the error is already filed and need to be fixed then it will be good to have `CONSOLE_ERRORS_TO_FIX` only.

Reply:
this is not an issue, its just an error thrown because of #20189 and thats why I have put this in the CONSOLE_ERRORS_TO_IGNORE as fixing that will fix it too. and we will know that once #20189 is fixed we can remove it too.